lib/Conversion/CfToHandshake/CfToHandshake.cpp

```diff
@@ -274,7 +274,7 @@ LogicalResult LowerFuncToHandshake::matchAndRewrite(
 
 SmallVector<NamedAttribute>
 LowerFuncToHandshake::deriveNewAttributes(func::FuncOp funcOp) const {
-  SmallVector<NamedAttribute, 4> attributes;
+  SmallVector<NamedAttribute, 3> attributes;
```

Comment:
Why this change? Please revert it unless you have a good reason for it.

Reply:
There is a compilation error saying the return type of the function does not match the declared function return type. Thus, we changed this line from 4 to 3.